gammapy/spectrum/flux_point.py

```diff
@@ -1303,20 +1302,18 @@ def residuals(self, method="diff"):
 
         model = self.model(fp.e_ref)
 
-        residuals = data - model
-
```

Comment:
@adonath  I also remove this line in this PR since it was not used.

Reply:
OK